src/test/java/com/softserveinc/ita/dkrutenko/rozetkaCheckCartTest.java

```diff
@@ -0,0 +1,109 @@
+package com.softserveinc.ita.dkrutenko;
+
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.ITestResult;
+import org.testng.annotations.*;
+
+public class rozetkaCheckCartTest {
+    private final String URL= "https://rozetka.com.ua/";
+    private WebDriver driver;
+    private WebDriverWait driverWait;
+
+    @BeforeSuite
+    public void beforeSuite() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Admin//Documents//GitHub//chromedriver.exe");
+    }
+
+    @BeforeClass
+    public void beforeClass() {
+        //setting up new Chrome driver
+        driver = new ChromeDriver();
+        //page load time
+        driver.manage().timeouts().pageLoadTimeout(10, TimeUnit.SECONDS);
+        //wait amout of time before it throws "No such Element Exception
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        //Maximize chrome window
+        driver.manage().window().maximize();
+    }
+    //implicitly wait for 2 seconds and quit the driver(close chrome window)
+    @AfterClass(alwaysRun = true)
+    public void afterClass() {
+        driver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
+        driver.quit();
+    }
+    //load google.com and delete cookies before work
+    @BeforeMethod
+    public void beforeMethod() {
+        //Launching actual url
+        driver.get(URL);
+        driver.manage().deleteAllCookies();
+    }
+    //print some message after test finish
+    @AfterMethod
+    public void afterMethod(ITestResult result) {
+        if(!result.isSuccess()) {
+            String testName = result.getName();
+            System.out.println("Congratulations, " + testName + " has finished working.");
```

Comment:
sysout's not allowed in PRs, this is noted in the convention

Reply:
Sysouts was for me and to show result after test. I know that it is not in convention... Will fix this comments soon